### ISSUE TITLE
The attributes of image is determined to Sentence

### ISSUE THREAD

**Issue by kongou-ae** [redpen-cc/redpen#682]
The attributes of image is not sentence. But RedPen determines it sentence. 

## asciidoc

```
image:images/logo.png[Company Logo]

image:images/logo.png["Company Logo",height=32]

image:screen-thumbnail.png[height=32,link="screen.png"]

image::images/logo.png[scaledwidth="75%",alt="Company Logo"]

image::images/tiger.png["Tiger image",align="left"]
```

## Validator

```
function validateSentence(sentence) {
  console = {
      log:print
  };
  console.log(sentence.content)
}
```

## Result

```
Company Logo
"Company Logo",height=32
height=32,link="screen.png"
scaledwidth="75%",alt="Company Logo"
"Tiger image",align="left"
```

**Comment by takahi-i** [comment]
Thanks @kongou-ae as always! I will look into this behavior.

**Comment by takahi-i** [comment]
Hello @kongou-ae. I fixed this issue in #686. Please let me know if you have the problem.

**Comment by kongou-ae** [comment]
Thanks!
